src/content/1.7/development/page-reference/back-office/product/form/_index.md

```diff
@@ -0,0 +1,254 @@
+---
+title: The Product Form
+menuTitle: Product Form
+weight: 42
+---
+
+# The New Product Form
+{{< minver v="1.7.8" title="true" >}}
+
+In most recent Product page, which was remade in 1.7.8, the form is divided into multiple sub forms. Each form has dedicated CQRS commands to manage its content.
+For previous versions you can read this documentation about [product form before 1.7.8]({{< ref "1.7/development/page-reference/back-office/product/form/product-form-before-1-7-8" >}}).
+
+## Creation / Update commands
+
+Except for the initial save, all product modifications are **updates**, which is why `AddProductCommand` is a special case. It requires only:
+
+- the product type
+- localized names (only the name in default language is required)
+
+Once the product is created and its ID has been generated most of the other CQRS commands are **updates** so they will need a `ProductId` identifier.
+They allow **partial updates** to minimize the amount of sent data, of course even partial update require a minimum of consistency and will have validation rules such as required fields in some cases.
+
+## Product form
+
+This form combines different sub form types, each one is handled by dedicated CQRS commands.
+
+| Form type                | Field      | Description                                              | CQRS Command                                           |
+|:-------------------------|:-----------|:---------------------------------------------------------|:-------------------------------------------------------|
+| **BasicInformationType** | `basic` | Contains the basic product information of the Product | `AddProductCommand` and `UpdateBasicInformationCommand` |
+| **DescriptionType** | `description`    | Contains the description of the Product | `UpdateDescriptionCommand` |
+| **ShortcutType** | `shortcut`    | Contains shortcut for prices and quantity of the Product | `UpdatePricesCommand`, `UpdateStockCommand` and `UpdateOptionsCommand` |
+| **RelationshipsType**    | `relationships` | Contains relationships of the product | N/A |
+| **ShortcutType**         | `shortcuts`     | Contains shortcut fields editable in other sub forms but accessible in first tab easily | N/A |
+| **TypeaheadProductPackCollectionType** | `pack_items` | List of products (for Pack of product) | `AddProductToPackCommand` `UpdateProductPackCommand` `RemoveProductFromPachCommand` |
+| **ProductCategoriesType** | `product_categories` | Manage product categories | N/A |
+
+### BasicInformationType
+
+| Fields | Field type                       | Description            |
+|:-------|:---------------------------------|:-----------------------|
+| `product_id` | `HiddenType` | The *Product* ID |
```

Comment:
whats the idea behind product_id here?
after we save a ghost product for first time, we fill this value up?
shouldn't we redirect to edit page after we save the ghost product instead and have an id from request?

Reply:
You're right the id in the request/url is enough, I dropped this one as well